apps/omg_eth/test/omg_eth/eth_test.exs

```diff
@@ -91,7 +92,14 @@ defmodule OMG.EthTest do
 
     {:ok, height} = Eth.get_ethereum_height()
 
-    assert {:ok, [%{amount: 1, blknum: 1, owner: contract.authority_addr, currency: @eth, eth_height: height}]} ==
-             Eth.RootChain.get_deposits(1, height, contract.contract_addr)
+    authority_addr = contract.authority_addr
+    deposits = Eth.RootChain.get_deposits(1, height, contract.contract_addr)
+
+    assert {:ok, [%{amount: 1, blknum: 1, owner: ^authority_addr, currency: @eth, eth_height: height, log_index: 0}]} =
+             deposits
+
+    {:ok, [%{} = deposit]} = deposits
```

Comment:
I think you should be able to get the txhash from either `RootChainHelper.deposit` or `transact_sync!` I'm just not 100% sure which would be nicer. Then you can assert `root_chain_txhash` is the right thing. Gives you a tiny bit more of coverage and shorter test, I think warranted here (this here is a smoke test of the `Eth` plumbing using a couple of simple contract calls, so seems to fit well)